lib/diversity/engine.rb

```diff
@@ -31,6 +31,11 @@ def initialize(options = {})
       @debug_level = 0
       @logger = @options[:logger]
 
+      @cache = Moneta.build do
+        use :Expires, expires: 3600
+        adapter :Memory
```

Comment:
The LRUHash isn't compatible with Expires.  Expires takes the value and places it in an array with a timestamp, and the LRUHash craches on the array not having bytesize :P